test/test_helper.rb

```diff
@@ -2,10 +2,12 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'mocha/mini_test'
+require 'capybara/rails'
 
 Dir["#{Rails.root}/test/support/**/*"].each { |file| require file }
 
 class ActiveSupport::TestCase
   # Add more helper methods to be used by all tests here...
   include FactoryGirl::Syntax::Methods
+  include Capybara::DSL
```

Comment:
We don't need `Capybara::DSL` for all tests. The module is only included for [accpetance tests](https://github.com/ruby-bench/ruby-bench-web/blob/1e3413a61eb5aa8823d7383425a9c10a84e81d47/test/acceptance/test_helper.rb)